testing/infra/terraform/modules/soaktest_workers/README.md

```diff
@@ -0,0 +1,52 @@
+## Soaktest worker module
+
+This module sets up worker with load generation binary for soaktest configured as a systemd unit along with required monitoring setup. The workers generated are immutable in nature and are recreated for every configuration change.
```

Comment:
Aren't we missing the `header.md` file in this PR?

Reply:
Sorry, I am not sure where the requirement of `header.md` comes from. Is it required as part of a terraform module?

Reply:
How did you generate this `README.md` file?

In the makefile, there is a target called `tf-docs`. When running it, it will use the contents of the `header.md file to generate the `README.md` file. If that file isn't checked in, then the contents will be re-generated in the future and we'll lose this information.

https://github.com/elastic/apm-server/blob/3de716555254975194ad5e6729e3cbf5b4536b88/Makefile#L129-L133

Reply:
Ah I see. I used the [terraform-docs cli](https://github.com/terraform-docs/terraform-docs) directly, wasn't aware about the Makefile. Let me update.